gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineTestingIntegrationTest.groovy

```diff
@@ -79,4 +78,30 @@ class BaselineTestingIntegrationTest extends AbstractPluginTest {
         new File(projectDir, "build/reports/tests/test/classes/test.TestClass4.html").exists()
         new File(projectDir, "build/reports/tests/test/classes/test.TestClass5.html").exists()
     }
+
+    def 'runs integration tests with junit5'() {
+        when:
+        buildFile << '''
+        plugins {
+            id 'org.unbroken-dome.test-sets' version '2.1.1'
+        }
+        '''.stripIndent()
+        buildFile << standardBuildFile
+        buildFile << '''
+
+        testSets {
+            integrationTest
+        }
+        
+        dependencies {
+            integrationTestImplementation "org.junit.jupiter:junit-jupiter:5.4.2"
```

Comment:
I just double checked and this should keep the `junit-jupiter-engine` as runtime only. https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter/5.5.1